includes/payment-methods/class-wc-stripe-payment-request.php

```diff
@@ -1969,11 +1969,32 @@ protected function build_display_items( $itemized_display_items = false ) {
 			];
 		}
 
+		$calculated_total = apply_filters_deprecated(
+			'woocommerce_stripe_calculated_total',
+			[ WC_Stripe_Helper::get_stripe_amount( $order_total ), $order_total, WC()->cart ],
+			'9.6.0',
+			'wc_stripe_calculated_total',
+			'The woocommerce_stripe_calculated_total filter is deprecated since WooCommerce Stripe Gateway 9.6.0, and will be removed in a future version. Use wc_stripe_calculated_total instead.'
+		);
+
+		/**
+		 * Filters the calculated total for the order.
+		 *
+		 * @since 9.6.0
+		 *
+		 * @param float $calculated_total The calculated total from the deprecated filter.
+		 * @param float $order_total The order total.
+		 * @param WC_Cart $cart The cart object.
+		 *
+		 * @return float The calculated total.
+		 */
```

Comment:
Given that we are using the same filter and arguments, we should reference the existing filter PHPdoc rather than duplicating the docs as per the WordPress core docs: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-1-duplicate-hooks
```suggestion
		/**
		 * This filter is documented in includes/payment-methods/class-wc-stripe-express-checkout-helper.php
		 */
```